<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="field" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>400</height>
       </size>
      </property>
      <widget class="Line" name="goal1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>10</width>
         <height>200</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="midline">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>20</width>
         <height>401</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="goal2">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>100</y>
         <width>10</width>
         <height>200</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QWidget" name="playerlist" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>70</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QComboBox" name="teamstate_2">
         <property name="currentText">
          <string>Spectator</string>
         </property>
         <item>
          <property name="text">
           <string>Spectator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="teamstate_3">
         <property name="currentText">
          <string>Spectator</string>
         </property>
         <item>
          <property name="text">
           <string>Spectator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QComboBox" name="teamstate_4">
         <property name="currentText">
          <string>Spectator</string>
         </property>
         <item>
          <property name="text">
           <string>Spectator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPlainTextEdit" name="playername2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextEditable</set>
         </property>
         <property name="placeholderText">
          <string>Player 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="playername1">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextEditable</set>
         </property>
         <property name="placeholderText">
          <string>Player 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPlainTextEdit" name="playername3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextEditable</set>
         </property>
         <property name="placeholderText">
          <string>Player 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="teamstate_1">
         <property name="currentText">
          <string>Spectator</string>
         </property>
         <item>
          <property name="text">
           <string>Spectator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPlainTextEdit" name="playername4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextEditable</set>
         </property>
         <property name="placeholderText">
          <string>Player 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
